Assets/Scripts/UI/LogText/Injury.cs

```diff
@@ -36,11 +38,54 @@ public override IEnumerable<LogRecord> Convert(Ffb.Dto.Reports.Injury report)
             if (report.armorRoll?.Length > 0)
             {
                 yield return new LogRecord($"<b>Armour Roll [ {report.armorRoll[0]} ][ {report.armorRoll[1]} ]</b>");
+                yield return new LogRecord($"<b>Rolled Total of {report.armorRoll[0] + report.armorRoll[1]}</b>");
 
-                //
-                // TODO: Incomplete
-                //
-            }
+                int armorModifierTotal = 0;
+                int i = 0;
+
+                var status = "";
+                int rolledTotal = report.armorRoll[0] + report.armorRoll[1];
+
+                foreach (FFBEnumeration entry in report.armorModifiers)
+                {
+                   var armourModifier = entry.AsArmorModifier();
+                    i = i | ((armourModifier == ArmorModifier.Claws ? 1 : 0));
+                    if (armourModifier.Modifier != 0)
+                    {
+                        armorModifierTotal += armourModifier.Modifier;
+
+                        status += armourModifier.Modifier > 0 ? " + " : " - ";
```

Comment:
Try to avoid string concatenation with +. See if you can find a way to use $"..." instead

Reply:
I'm not sure what the best way to do that in a foreach loop. It seems to be building up the status in each iteration - I don't know what the best practice is there. i will take a look